tests/test_api.py

```diff
@@ -68,5 +68,5 @@ def test_API_with_extra_parameters():
     client.show_header.should.be.true
     client.proxies.should.equal(proxies)
     client.session.headers.should.have.key("Authorization").which.should.equal(
-        f"X-NVRST-APIKEY {api_key}"
+        f"X-DTMX-APIKEY {api_key}"
     )
```

Comment:
@martinkersner We need to update our User API for this change right?

Reply:
Yes https://github.com/Bisonai/neverest/issues/499, but the python package is not yet officially published. Once we update API, we can publish the package as well.